src/classes/LightningMetadataController_Tests.cls

```diff
@@ -0,0 +1,34 @@
+@isTest
```

Comment:
Element `@isTest
private class LightningMetadataController_Tests {

    @isTest
    static void getSObjectMetadataTest() {
        String sobjectName = 'Account';
        DescribeSObjectResult sobjectDescribe = Schema.Account.SObjectType.getDescribe();

        Test.startTest();
        LightningMetadataController.SObjectMetadata sobjectMetadata = LightningMetadataController.getSObjectMetadata(sobjectName);
        Test.stopTest();

        System.assertEquals(sobjectName, sobjectMetadata.name);
        ` does not comply with naming convention. Please rename.